static/app/views/dashboardsV2/widget/metricWidget/fields.tsx

```diff
@@ -0,0 +1,101 @@
+import {SelectValue} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {FieldValue, FieldValueKind} from 'sentry/views/eventsV2/table/types';
+
+export type MetricsColumnType = 'set' | 'counter';
+
+export enum FieldKey {
+  SESSION = 'session',
+  USER = 'user',
```

Comment:
```suggestion
  SESSION = 'sentry.sessions.session',
  USER = 'sentry.sessions.user',
```

Reply:
should we use the fields defined in this file?

https://github.com/getsentry/sentry/blob/eb5781c217833acf605cbf4ec7de5e39b6756614/static/app/utils/metrics/fields.tsx#L5